octopus/Dockerfile.tentacle

```diff
@@ -0,0 +1,8 @@
+FROM octopusdeploy/tentacle:8.1.1693
```

Comment:
Non-blocking: the newest version is 8.1.1838. These update pretty frequently, but I am also fine leaving it as-is for now. We'll have to come up with an upgrade strategy if/when we ever onboard STLT users to this.

Reply:
I can push this update along with a name update related to your other comment